### PR TITLE
chore: apply CI for clang-format only for dash files

### PR DESCRIPTION
**Pull request by knst**
## Issue being fixed or feature implemented
**Note**: should be this PR either https://github.com/dashpay/dash/pull/5942 be merged, not both

CI clang-format triggers to non-dash files + clang format is differ from out current formatting.

## What was done?
See each commits

## How Has This Been Tested?
See CI result
To test locally how new style will look, just run this command:
```
diff -u <(cat {coinjoin,governance,llmq,evo,masternode}/*.{h,cpp}) <(clang-format-16 {coinjoin,governance,llmq,evo,masternode}/*.{h,cpp} ) 
```

## Breaking Changes
N/A

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/.clang-format [hunk 1]
```diff
@@ -4,31 +4,33 @@ AlignAfterOpenBracket: true
...
 AlwaysBreakTemplateDeclarations: true
 BinPackArguments: true
 BinPackParameters: false
-BreakBeforeBinaryOperators: false
+BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Custom
 BraceWrapping:
   AfterClass: true
   AfterFunction: true
+  AfterEnum: true
```
Comment (UdjinM6) [inline src/.clang-format:21]: not sure I like this one... any specific reason why we'd want this?

**Comment by knst** [comment]
without this one, new clang config is trying to contact lines of enums like that:
```
 // FOR SEEN MAP ARRAYS - GOVERNANCE OBJECTS AND VOTES
-enum class SeenObjectStatus {
-    Valid = 0,
-    ErrorInvalid,
-    Executed,
-    Unknown
-};
+enum class SeenObjectStatus { Valid = 0, ErrorInvalid, Executed, Unknown };
``` 
and
```
-enum class GovernanceObject : int {
-    UNKNOWN = 0,
-    PROPOSAL,
-    TRIGGER
+enum class GovernanceObject : int { UNKNOWN = 0, PROPOSAL, TRIGGER };
```

that's not a single example.

But I am open for proposes how to avoid this suggestion by the better way.
`+enum class GovernanceObject : int { UNKNOWN = 0, PROPOSAL, TRIGGER };` is too bad IMO

**Comment by UdjinM6** [comment]
pls check d1a63111a3

**Review by UdjinM6 — APPROVED** [review]
utACK

**Comment by knst** [comment]
as pasta noticed:
```
     CPendingDsaRequest(CService addr_, CCoinJoinAccept dsa_) :
-        addr(std::move(addr_)),
-        dsa(std::move(dsa_)),
-        nTimeCreated(GetTime())
+        addr(std::move(addr_)), dsa(std::move(dsa_)), nTimeCreated(GetTime())
     {
     }
```
that's not expecting changes

**Comment by UdjinM6** [comment]
a couple more suggestions 8b3b80001e

**Review by UdjinM6 — APPROVED** [review]
light ACK, not 100% accurate but much better than before imo

**Review by PastaPastaPasta — APPROVED** [review]

**Pull request merged**